Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -628,7 +675,7 @@ extension ProjectDescription.DeploymentTargets {
         packageName _: String
     ) throws -> Self {
         if type == .macro {
-            return .macOS(minDeploymentTargets[.macOS] ?? oldestVersions(for: Version(5, 9, 0)).macOS!)
+            return .macOS("14.0")
```

Comment:
This probably should change? maybe? i was getting warnings when hacking on it so I'm not entirely sure what we want long term, but setting to 14 works.